issues/002-shared-kernel-naming.md

```
@@ -0,0 +1,32 @@
+# 共通層の命名をDDD用語に合わせる
+
+## 概要
+現在の `common/` ディレクトリ名をDDDの標準用語である「Shared Kernel」に合わせることを検討。
+
+## 現状
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)